VideoAPI/VideoApi.UnitTests/Mappings/ConferenceToDetailsResponseMapperTests.cs

```diff
@@ -40,5 +44,17 @@ public void should_map_all_properties()
                 .Excluding(x => x.CurrentRoom)
             );
         }
+
+        bool ExcludeIdFromMessage1(IMemberInfo member)
```

Comment:
please remove whichever method is unused

Reply:
Done